gourmet/plugins/import_export/mealmaster_plugin/mealmaster_importer.py

```diff
@@ -539,8 +539,8 @@ def find_columns (strings, char=" "):
     # we start with the columns in the first string
     if not strings:
         return None
-    strings=strings[0:]
-    strings.sort(lambda x,y: len(x)>len(y))
+    strings=list(strings)
+    strings.sort(key=len, reverse=True)
```

Comment:
I'm not sure about this one either, as the original doesn't even return -1, 0, 1